licenses/management/commands/load_html_files.py

```diff
@@ -227,26 +226,13 @@ def handle(self, input_directory, **options):
                     "Content-Transfer-Encoding": "8bit",
                 }
 
+                # Use the English message text as the message key
                 for internal_key, translation in messages_text.items():
                     if language_code == "en":
                         message_key = translation.strip()
                         message_value = ""
                     else:
-                        # WORKAROUND - by-nc-nd 4.0 NL has an extra item under s3a.
-                        if (
-                            internal_key == "s3a4_if_you_share_adapted_material"
-                            and internal_key
-                            not in english_by_license_code[license_code]
-                        ):
-                            message_key = (
-                                "If You Share Adapted Material You produce, the Adapter's "
-                                "License You apply must not prevent recipients of the Adapted "
-                                "Material from complying with this Public License."
-                            )
-                        else:
-                            message_key = english_by_license_code[license_code][
-                                internal_key
-                            ]
+                        message_key = english_messages[internal_key]
```

Comment:
`KeyError: 'license_long'`

Reply:
This key error seems to only be generated if I run `load_html_files` without the `--versions` argument.

Reply:
```
python3.8 manage.py load_html_files --versions=1.0,3.0 ../creativecommons.org/docroot/legalcode/ 
```
Works as expected. I think the above error is probably tied to 4.0 licenses.
```
python3.8 manage.py load_html_files --versions=4.0 ../creativecommons.org/docroot/legalcode/
```
If I run 4.0 licenses by themselves; this error is generated again.
`KeyError: 's3a4_if_you_share_adapted_material'`

Reply:
I'll revert this change. After that, ```load_html_files ../creativecommons.org/docroot/legalcode/``` runs without error for me.